index.js

```diff
@@ -1,71 +1,91 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+console.log("%cQuestion 1", "color: red") 
+let  counter = 1 
 
+counter = 2
+console.log(counter) 
+// => 2
+console.log("%c----------", "color: red") 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red") 
+const name = "Linda"
 
-// console.log(name)
+console.log(name)
 // => "Raffy"
 // name = "Not Raffy" 
 // => TypeError
-// console.log("%c----------", "color: red") 
-
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
+function drinkWater(x) {
+  console.log("Man I sure am thirsty")
+  console.log("Ahh that hits the spot")
+  console.log( x - 1)
+}
```

Comment:
Great! Although I would not encourage the use of single letter variable names! I am not sure what x is supposed to represent. In this case I would prefer something like `currentThirstLevel` or `thirstLevel`

Reply:
Also you should return the value of the thirstLevel at the end of the function

```
function drinkWater(thirstLevel) {
  console.log("Man I sure am thirsty");
  console.log("Ahh that hits the spot");
  return thirstLevel - 1;
}
```